src/components/landing/Nav.tsx

```diff
@@ -0,0 +1,76 @@
+import styled, { css } from 'styled-components';
+import { icDeliverbleBlue, icDeliverbleWhite } from 'public/assets/icons';
+import { COLOR } from 'src/styles/color';
+import { FONT_STYLES } from 'src/styles/fontStyle';
+import ImageDiv from '../common/ImageDiv';
+import Link from 'next/link';
+
+interface NavProps {
+  isFirstScrolled?: boolean;
+  isSecondScrolled?: boolean;
+}
+
+function Nav(props: NavProps) {
+  const { isFirstScrolled = false, isSecondScrolled = false } = props;
+
+  return (
+    <StNav isSecondScrolled={isSecondScrolled}>
+      {isFirstScrolled ? (
+        <ImageDiv src={icDeliverbleBlue} className="logo" layout="fill" alt="" />
+      ) : (
+        <ImageDiv src={icDeliverbleWhite} className="logo" layout="fill" alt="" />
+      )}
```

Comment:
```suggestion
      <ImageDiv src={isFirstScrolled ? icDeliverbleBlue : icDeliverbleWhite} className="logo" layout="fill" alt="" />
```

이렇게 한 줄로 줄일 수 있습니다 ㅎㅎ

Reply:
헐 모야 이렇게 src 바꾸니 완전 좋네여 참고해야지